### PR TITLE
updated URLs as per the services change

### PR DESCRIPTION
**Pull request by Lokendra-egov**

### CODE REVIEW

File: frontend/micro-ui/web/src/setupProxy.js [hunk 1]
```diff
@@ -29,17 +29,17 @@ module.exports = function (app) {
     '/inbox/v2/_search',
     '/individual',
     '/org-services',
-    '/wms/contract/_search',
+    '/wms/mukta-contract/_search',
```
[the remark this answers is not in the document]
Comment (Lokendra-egov) [reply frontend/micro-ui/web/src/setupProxy.js:32]: yes, that's a service name and service name has been changed from contract to mukta-contract
@jagankumar-egov

File: frontend/micro-ui/web/micro-ui-internals/example/src/setupProxy.js [hunk 2]
```diff
@@ -74,23 +74,23 @@ module.exports = function (app) {
     "/inbox/v2/_search",
     "/individual",
     "/org-services",
-    "/wms/contract/_search",
+    "/wms/mukta-contract/_search",
```
[the remark this answers is not in the document]
Comment (Lokendra-egov) [reply frontend/micro-ui/web/micro-ui-internals/example/src/setupProxy.js:77]: I've registered these URLs in the action-test.json file as well.

File: frontend/micro-ui/web/micro-ui-internals/example/src/setupProxy.js [hunk 3]
```diff
@@ -74,23 +74,23 @@ module.exports = function (app) {
     "/inbox/v2/_search",
     "/individual",
```
Comment (Tulika-eGov) [inline frontend/micro-ui/web/micro-ui-internals/example/src/setupProxy.js:75]: Check for this as well @Lokendra-egov if the name is right?

**Review by Tulika-eGov — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/example/src/setupProxy.js (hunk 3 above)
Comment (Lokendra-egov) [reply frontend/micro-ui/web/micro-ui-internals/example/src/setupProxy.js]: resolved

File: frontend/micro-ui/web/src/setupProxy.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Lokendra-egov) [reply frontend/micro-ui/web/src/setupProxy.js:32]: yes, but we're not making any changes in wms so we're simply calling mukta-contract through wms in our environment only but we're not having wms in our environment.

**Review by egov-arindam — APPROVED** [review]

**Pull request merged**